examples/display_lvgl_demos/main/CMakeLists.txt

```diff
@@ -1,17 +1,3 @@
-set(LV_DEMO_DIR "")
-set(LV_DEMOS_SOURCES "")
-if(CONFIG_LV_USE_DEMO_BENCHMARK)
-    list(APPEND LV_DEMO_DIR ../managed_components/lvgl__lvgl/demos)
-    file(GLOB_RECURSE LV_DEMOS_SOURCES ${LV_DEMO_DIR}/*.c)
-endif()
```

Comment:
Why these lines were removed? Now, the `lv_demo_benchmark` cannot be compiled.

Reply:
will check, thanks

Reply:
@espzav I had to increse size of `factory` partition so the benchmark demo would fit. But apart from that it works OK. I'd like to use the files from LVGL's CMake. https://github.com/lvgl/lvgl/blob/master/env_support/cmake/esp.cmake#L32

What issues you see in the lv_demo_benchmark demo?

Reply:
Thank you :-)